server/src/main/resources/db/common/V73__alter_column_naming_convention.sql

```diff
@@ -0,0 +1,58 @@
+-- Rename columns in table feedback_requests
+
+ALTER TABLE feedback_requests
+RENAME COLUMN creatorId TO creator_id;
+
+ALTER TABLE feedback_requests
+RENAME COLUMN requesteeId TO requestee_id;
+
+ALTER TABLE feedback_requests
+RENAME COLUMN recipientId TO recipient_id;
+
+ALTER TABLE feedback_requests
+RENAME COLUMN templateId TO template_id;
+
+ALTER TABLE feedback_requests
+RENAME COLUMN sendDate TO send_date;
+
+ALTER TABLE feedback_requests
+RENAME COLUMN dueDate TO due_date;
+
+ALTER TABLE feedback_requests
+RENAME COLUMN submitDate TO submit_date;
+
+
+-- Rename columns in table feedback_request_questions
+
+ALTER TABLE feedback_request_questions
+RENAME COLUMN requestId to request_id;
+
+ALTER TABLE feedback_request_questions
+RENAME COLUMN questionContent TO question;
+
+ALTER TABLE feedback_request_questions
+RENAME COLUMN orderNum TO question_number;
+
+
+-- Rename columns in table feedback_templates
```

Comment:
can we include V72 migrations in here so all feedback_templates are together ? In that case you will not need V72

Reply:
Good idea, the migrations have been merged together into V72 in commit `e8c6e66`